src/examples/talos.cpp

```diff
@@ -0,0 +1,79 @@
+#include <algorithm>
```

Comment:
We should either remove completely the `talos` example or make it self-containable, that is, include the meshes within `robot_dart`.

Reply:
Maybe we can have a submodule pointing to [talos_robot](https://github.com/pal-robotics/talos_robot) ?

Reply:
I suggest to remove the talos example for now, as we will add a few robots to be officially supported by robot_dart in the future. Can you make an example with the KUKA arm for the new functionalities?